### PR TITLE
Allow defining multiple containers in a Pod

### PR DESCRIPTION
**Pull request by carlossg**
Based on #64 adding easier migration from 0.8

### CODE REVIEW

**Comment by sdouche** [comment]
Hello. It would be cool to add a valid example of a multiple containers pod to build something (a container with JNLP and another one with the project to build) w/o pipeline.

**Comment by carlossg** [comment]
There is an example in #68 with pipeline.
Without pipeline, I guess you could install kubectl in the slave container and run `kubectl exec` into the other containers, or run services in the other containers and access them through tcp/ip using `localhost`

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java [hunk 1]
```diff
@@ -222,50 +260,97 @@ public void setImagePullSecrets(List<PodImagePullSecret> imagePullSecrets) {
...
+            this.containers.addAll(items);
+        }
+    }
+
+    @Nonnull
     public List<ContainerTemplate> getContainers() {
         return containers;
     }
 
+    @SuppressWarnings("deprecation")
```
Comment (iocanel) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java]: So, if I get it right, this is the proper way to create the fallback container, right?

Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java]: yes, this will read the xml and convert it to the new format

**Comment by shanemcd** [comment]
This is great news. I'm currently in the process of building out a new CI infrastructure and would love to use this. Any idea when 0.9 is going to be released?

**Comment by JeanMertz** [comment]
Interested in this as well, rally nice 🎉. Any idea when we might expect 0.9 to ship?

**Pull request merged**